gto-support-db-coroutines/src/main/kotlin/org/ccci/gto/android/common/db/CoroutinesFlowDao.kt

```diff
@@ -1,17 +1,36 @@
 package org.ccci.gto.android.common.db
 
 import androidx.annotation.AnyThread
-import androidx.lifecycle.asFlow
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.channels.awaitClose
+import kotlinx.coroutines.channels.trySendBlocking
+import kotlinx.coroutines.flow.callbackFlow
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.flow.map
+
+interface CoroutinesFlowDao : CoroutinesDao, Dao {
+    @OptIn(ExperimentalCoroutinesApi::class)
+    private fun invalidationFlow(types: Collection<Class<*>>) = when {
```

Comment:
this method produces a Flow of classes that have been invalidated that match the `types` specified.

so, `invalidationFlow(Model::class.java)` will emit `Model::class.java` every time there is a db update made to the table representing `Model::class.java`. If there is a change to `Model2::class.java` it will not emit a value because we aren't monitoring that type for invalidations.